### ISSUE TITLE
Suggest is only for local users

### ISSUE THREAD

**Issue by hoergen** [friendica/friendica#5176]
### Expected behavior
Suggest should work as well for "foreign" users on an other node
Think this is a friendica feature? Or maybe Diaspora as well?

### Actual behavior
Only able to suggest contacts to local users

### Steps to reproduce the problem
Open a contact that is not on your node and try to find the menu entry on the upper right corner (frio)

### Friendica version you encountered the problem
Friendica develop / git / php7.0 / 5.5.5-10.0.32-MariaDB-0+deb8u1

**Comment by tobiasd** [comment]
For me this is possible in any theme I've tested, including frio. It is in the action menu for the contact.

**Comment by MrPetovan** [comment]
Even Diaspora contacts?

**Comment by hoergen** [comment]
hmm I treid it with diaspora and it didnt work.

**Comment by MrPetovan** [comment]
My understanding is that, at the moment you can:
1. suggest only Friendica contacts, local or remote
2. only to Friendica contacts, local or remote

Only the 1. could be changed as there's no standard way of suggesting contacts across multiple protocols.

**Comment by MrPetovan** [comment]
I'm confused about who you can suggest to whom.

**Comment by hoergen** [comment]
Oh ok.
I as friendica user want send suggestions to another friendica user on my node.

In that scenario I would think that I can suggest every account that is in my list.

**Comment by MrPetovan** [comment]
Including foreign network accounts?

**Comment by tobiasd** [comment]
Yes including foreign network accounts. Non local friendica accounts can be suggested to contacts already.
